collect_app/src/test/java/org/odk/collect/android/utilities/ArrayUtilsTest.java

```diff
@@ -0,0 +1,41 @@
+package org.odk.collect.android.utilities;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class ArrayUtilsTest {
+    @Test
+    public void toPrimitive() throws Exception {
+        Long[] object = new Long[] {1L, 2L, 3L};
+        long[] primitive = new long[] {1, 2, 3};
+        assertArrayEquals("toPrimitiveTest", primitive, ArrayUtils.toPrimitive(object));
+
+        long[] emptyLongPrimitive = new long[0];
+        assertArrayEquals("toPrimitiveNullTest", emptyLongPrimitive, ArrayUtils.toPrimitive(null));
```

Comment:
How about improving your first argument: "toPrimitive(null) did not produce an empty long array"